src/aggregate_utils.py

```diff
@@ -1,17 +1,39 @@
-from src.download_utils import get_default_params, download_store_data
+from download_utils import get_default_params, download_store_data
+import re
 
 
 def extract_upcoming_promotions(store_data):
+    """
+    Extract upcoming promotions from store data.
+
+    args: store_data: dict
+
+    Returns: List of upcoming promotions.
+    """
     promos = store_data["elements"]
```

Comment:
This is a minor remark. Codacy Static Code Analysis mentions a minor issue with PEP 257 docstring style checker:
> Multi-line docstring summary should start at the first line (D212)

That is the style which is most common. However, it is true that [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) allows both styles.
> The summary line may be on the same line as the opening quotes or on the next line.